packages/dds/tree/src/simple-tree/index.ts

```diff
@@ -121,6 +128,7 @@ export {
 	type InsertableField,
 	type Insertable,
 	type UnsafeUnknownSchema,
+	normalizeAllowedTypes,
```

Comment:
I left a comment in the root index.ts file as well, but I don't think this function should be needed anywhere outside of the `simple-tree` directory.

Reply:
We discussed offline how normalizeAllowedTypes is being used for extra typechecks and error mitigation within agentEditReducer which is also used to provide the LLM with a feedback loop.